src/python_framework/nhd_reach_parallel_test.py

```diff
@@ -203,22 +208,52 @@ def compute_network_parallel_opportunistic(
     if debuglevel <= -1:
         for po, l in ordered_reaches.items():
             print(f'{po}: {len(l)}')
-    
+
+#command line input order: verbose,debuglevel,showtiming,supernetwork    
+def _handle_args():
+  parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+  parser.add_argument('--debuglevel',
+                      help='Set the debuglevel',
+                      dest='debuglevel',
+                      default=0)
+  parser.add_argument('--verbose',
+                      help='Set tHE verbose - leave blank for False',
+                      dest='verbose',
+                      type=bool,
+                      default=False)
+  # TODO: improve to be more intelligent about the argument to accept and making it a Path (argparse Action perhaps)
+  parser.add_argument('--showtiming',
+                      #help='Change the base directory when using SSL certificate and key files with default names',
+                      help='Set the showtiming - leave blank for False',
+                      dest='showtiming',
+                      type=bool,
+                      default=False)
+  parser.add_argument('--supernetwork',
+                      help='List of supernetworks (Pocono_TEST1,LowerColorado_Conchos_FULL_RES,Brazos_LowerColorado_ge5,Brazos_LowerColorado_FULL_RES,Brazos_LowerColorado_Named_Streams,CONUS_ge5,Mainstems_CONUS,CONUS_Named_Streams,CONUS_FULL_RES_v20',
+                      # action='append',
+                      # nargs=1,
+                      dest='supernetworks_list',
+                      default='Brazos_LowerColorado_ge5')
+
+  # parser.prog = package_name

```

Comment:
Let's use the verbose block from the prior commit: 
```
    parser.add_argument('-v, --verbose',
                        help='Verbose output (leave blank for quiet output)',
                        dest='verbose',
                        action='store_true')
```

Reply:
Also, there are a few changes to the `supernetwork` and `showtiming` options that should be imitated.